staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -58,7 +58,7 @@ var (
 		kubectl logs nginx --all-containers=true
 
 		# Return snapshot logs from all containers in pods defined by label app=nginx
-		kubectl logs -lapp=nginx --all-containers=true
+		kubectl logs -l app=nginx --all-containers=true
```

Comment:
I'd like to see this being consistent across all commands, have you checked all the examples if they are similar?
/lgtm cancel

Reply:
Thanks for your comments.I took a look for all help information about Kubectl.This seems not to exist in others.